metadata-db-app/src/migrations/052_add_coll_ids_catalog_acls.clj

```diff
@@ -48,20 +48,4 @@
 (defn down
   "Migrates the database down from version 52."
   []
-  (println "migrations.052-add-coll-ids-catalog-acls down...")
```

Comment:
Daniel mentioned this could destroy data in the system making ACLs unrecoverable, so we agreed it should be removed.